cas-server-core/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -118,15 +118,7 @@ public void setIssueAsynchronousCallbacks(final boolean asyncCallbacks) {
      */
     @Override
     public List<LogoutRequest> performLogout(final TicketGrantingTicket ticket) {
-        final Map<String, Service> services;
```

Comment:
The problem I experienced here is that service ticket weren't removed from cache registry when TGT ticket was removed. Discussed this with mmoayyed and he agreed we should remove service tickets together with TGT. Look here:
https://github.com/Jasig/cas/issues/1183